### ISSUE TITLE
[REQUEST] Add option to disable hardware acceleration.

### ISSUE THREAD

**Issue by kemzops** [hydralauncher/hydra#181]
Im having high CPU usage issues, i cant even navigate through the UI, I had similar issues before with electron applications usually disabling hardware acceleration helps, I saw another issue about high cpu usage but i want to leave this here:

https://www.electronjs.org/docs/latest/tutorial/offscreen-rendering
(also having CLI Flags wont hurt)

Thanks !.

**Comment by kemzops** [comment]
also having this option fixes:
```
[24096:0503/204853.056036:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[24096:0503/205120.656249:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
```

**Comment by kemzops** [comment]
#133

**Comment by zamitto** [comment]
The CPU usage was fixed on 1.2.0. You should have no problem with this since then

**Comment by kemzops** [comment]
I still think adding the ability to disable hardware acceleration is important for users with integrated GPU's, No ?
